packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -170,6 +170,22 @@ def check_adminrouter_metrics():
         check_adminrouter_metrics()
 
 
+def test_metrics_diagnostics(dcos_api_session):
+    """Assert that Admin Router metrics on master are present."""
```

Comment:
comment should be updated to reflect that master and slaves are all getting tested here